src/facade/dragonfly_connection.cc

```diff
@@ -451,7 +451,8 @@ io::Result<bool> Connection::CheckForHttpProto(FiberSocketBase* peer) {
 void Connection::ConnectionFlow(FiberSocketBase* peer) {
   auto dispatch_fb = fibers::fiber(fibers::launch::dispatch, [&] { DispatchFiber(peer); });
   ConnectionStats* stats = service_->GetThreadLocalConnectionStats();
-  stats->num_conns++;
+  ++stats->num_conns;
+  ++stats->conn_received_cnt;
```

Comment:
Will num_conns always be equal to conn_reveiced_cnt ? If so why do we need both?

Reply:
num_conns tells you how many connections you currently have. So it also decreases.
conn_received_cnt is evergrowing count that only counts accepted connections.